cmds/core/yes/yes.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2017-2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"strings"
+)
+
+func main() {
+	flag.Parse()
+	args := flag.Args()
+	yOutput := "y"
```

Comment:
I think calling it yes is fine, not yOutput. Even y would be fine. Go tends to emphasize terse names :-)

Reply:
Very convenient :)!